packages/redot-stringify/package.json

```
@@ -2,17 +2,17 @@
   "name": "redot-stringify",
   "version": "0.5.0",
   "description": "Graphviz dot compiler for redot",
-  "main": "redot-stringify.js",
+  "type": "module",
+  "main": "index.js",
+  "types": "index.d.ts",
```

```suggestion
  "exports": "./index.js",
```

Export map hides files for users, which is easier to document and prevent internal things from leaking.
With that, `types` and `main` is no longer needed (except when the `.js` -> `.d.ts` doesn‘t work)